src/olympia/addons/serializers.py

```diff
@@ -405,3 +407,35 @@ def fake_object(self, data):
                 obj.persona._broken = True
 
         return obj
+
+
+class ActivityLogSerializer(serializers.ModelSerializer):
```

Comment:
In a similar vein as my comment above: not should this should live in `addons/serializers.py`. I put the `File` and `Version` serializers there because they only make sense with the `Addon`, but I'm not so sure about the ones you're adding.